Student-Unity/src/main/java/csed/swe/studentunity/model/RevisionNotification.java

```diff
@@ -0,0 +1,24 @@
+package csed.swe.studentunity.model;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+@Entity(name = "RevisionNotification")
+public class RevisionNotification {
+    @Id
+    @Column(name = "user_id")
+    private Long id;
```

Comment:
what id is that ???????? 
tokenId or userId
if you mead userId, then the ``User user`` property is enough as it will refrence the user as a foreign key

Reply:
It's a foreign key refers to user id 
user property doesn't mean that this will save user as a field in this table it's a syntax of how you make foreign key in spring data jpa to map user_id field to id in user table

As you see in the screen shot down below revision notification table doesn't have user property
![Screenshot from 2022-12-16 22-59-26](https://user-images.githubusercontent.com/80472863/208188103-b48c1a25-1df3-4c01-8ede-2dfdd6cf239f.png)